plutus-core/untyped-plutus-core/testlib/Evaluation/Builtins/Integer/Common.hs

```diff
@@ -0,0 +1,98 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE TypeApplications    #-}
+{-# LANGUAGE TypeOperators       #-}
+module Evaluation.Builtins.Integer.Common
+where
+
+import Prelude hiding (and, not, or)
+
+import PlutusCore qualified as PLC
+import PlutusCore.Generators.QuickCheck.Builtin (AsArbitraryBuiltin (..))
+import PlutusCore.MkPlc (builtin, mkIterAppNoAnn, mkTyBuiltin, tyInst)
+
+import Evaluation.Builtins.Common
+
+import Test.QuickCheck (Gen, arbitrary)
+
+-- For property tests, we don't want to use the standard QuickCheck generator
+-- for Integers because that only produces values in [-99..99].  Here are some
+-- utilities to make it easier to use the better generator for
+-- AsArbitraryBuiltin Integer.
+type BigInteger = AsArbitraryBuiltin Integer
```

Comment:
They're not always big, but I couldn't think of a better name.

Reply:
I changed the generator to include some really big numbers so the name's a bit more accurate now.  It should really be something like `InterestingOrReallyBigInteger`, but wer'e not in Java-land.